sqlglot/dialects/mysql.py

```diff
@@ -325,6 +325,7 @@ class Parser(parser.Parser):
             "SCHEMA": exp.CurrentSchema.from_arg_list,
             "DATABASE": exp.CurrentSchema.from_arg_list,
             "STR_TO_DATE": _str_to_date,
+            "SUBSTRING_INDEX": exp.SubstringIndex.from_arg_list,
```

Comment:
You don't need to add this because it's automatically added, see [here](https://github.com/tobymao/sqlglot/blob/fb4122e80d1995bb87401e9ebe3749078c026a06/sqlglot/parser.py#L197) and [here](https://github.com/tobymao/sqlglot/blob/fb4122e80d1995bb87401e9ebe3749078c026a06/sqlglot/expressions.py#L7066-L7067).
```suggestion
```